src/components/track-tile/TrackTile.tsx

```diff
@@ -107,14 +98,19 @@ const TrackTileComponent = ({
 
   const goToTrackPage = useCallback(
     (e: GestureResponderEvent) => {
-      pushRouteWeb(permalink)
-      navigation.navigate('track', { id: track_id })
+      navigation.navigate({
+        native: { screen: 'track', params: { id: track_id } },
+        web: { route: permalink }
+      })
     },
-    [pushRouteWeb, permalink, navigation, track_id]
+    [navigation, permalink, track_id]
   )
 
   const goToArtistPage = (e: GestureResponderEvent) => {
-    navigation.navigate('profile', { handle: user.handle })
+    navigation.navigate({
+      native: { screen: 'profile', params: { handle: user.handle } },
+      web: { route: user.handle }
+    })
```

Comment:
I like this! My only thought is that there will be a fair amount of changes when we rip out web. But that should be fine

Reply:
yeah that is a really good call...